### DISCUSSION TITLE
Particular section(COLLAPSIBLE CARD) works well in Local but doesn't expands on github webpage

### DISCUSSION THREAD

**Discussion by amangupt01** [community/community#23295] [Other Feature Feedback, Questions, & Ideas]
<p>My webpage works well in local HTML file . But there is some problem in a section when I uploaded the page on github. The problem is actually with the file elements.html(‘PROJECTS’ section of the webpage ) . It is actually a COLLAPSIBLE PAPER CARD ( <a href="https://bbo-code.com/components/paper-collapse" title="Collapsible paper card" rel="nofollow noopener">https://bbo-code.com/components/paper-collapse</a> )</p>
<p>which expands when you click on it . But due to some reason I don’t know why the cards doesn’t expands on my webpage and the information inside the card is not shown at all. This works fine when I run the local HTML file. So please help me fix it .</p>
<p><a href="https://amangupt01.github.io/AmanGupta.io/elements.html" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/57101022/181076064-8c277b54-ee91-4308-aef9-78d9243bee3e.jpeg" alt="The_correct_way_that_works_on_local.jpg" data-base62-sha1="mE24Qd27bVpMnk2CTL0IpuDSEAV" width="369" height="230"> <img src="https://user-images.githubusercontent.com/57101022/181076066-46483177-2734-49c0-ac52-c16a8db6e73b.jpeg" alt="The_way_it_works_on_github_webpage_the_boxes_don't_expand.jpg" data-base62-sha1="krBYnX8lU3L8SfBqvrURBc1N8nC" width="690" height="435"><br>
</a></p>

**Comment by darrowby385** [comment, marked as answer]
<p>Hi, <a class="mention" href="/amangupt01">@amangupt01</a>!</p>
<p>This sort of trouble is commonly caused by incorrect file paths. A great way to diagnose these issues is using your browser’s development tools, specifically its console. Most browsers offer such functionality:</p>
<ul>
<li><a href="https://developers.google.com/web/tools/chrome-devtools/open" rel="nofollow noopener">Chrome DevTools</a></li>
<li>
<a href="https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console" rel="nofollow noopener">Firefox Web Console</a> </li>
<li>
<a href="https://support.apple.com/en-au/guide/safari/sfri20948/mac" rel="nofollow noopener">Safari Develop Menu</a> </li>
<li>
<a href="https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide/console" rel="nofollow noopener">Edge DevTools</a> </li>
</ul>
<p>Inspecting your site with my browser’s console, I spotted some helpful 404 errors, suggesting a CSS stylesheet named <span>paper-collapse.min.css</span> and a JavaScript file named paper-collapse.min.js were failing to load. Digging a little deeper, I noticed that these files were referenced as …/paper-collapse.min.css and …/paper-collapse.min.js in your HTML:</p>
<ul>
<li><a href="https://github.com/amangupt01/paper-collapse/blob/master/index.html#L12" rel="nofollow noopener">https://github.com/amangupt01/paper-collapse/blob/master/index.html#L12</a></li>
<li><a href="https://github.com/amangupt01/paper-collapse/blob/master/index.html#L100" rel="nofollow noopener">https://github.com/amangupt01/paper-collapse/blob/master/index.html#L100</a></li>
</ul>
<p>The …/ components of those URLs tell the browser to look for those files in a directory (folder) one level above that of the HTML file they are referenced in. However, we can see from your repository that the relevant files are actually in the <em>same</em> directory as the HTML file, not its parent directory. That is to say, all of these files are in the root of your repository:</p>
<p><a href="https://github.com/amangupt01/paper-collapse" rel="nofollow noopener">https://github.com/amangupt01/paper-collapse</a></p>
<p>I suspect this trouble arose because your index.html file was originally copied from the “demo” subdirectory, where these links would have been correct. You can resolve this trouble by removing the …/ components from the URLs in your HTML. For example, you could replace this line:</p>
<pre><code>&lt;link rel="stylesheet" href="../paper-collapse.min.css"&gt;
</code></pre>
<p>with this line:</p>
<pre><code>&lt;link rel="stylesheet" href="paper-collapse.min.css"&gt;
</code></pre>
<p>I hope that helps!</p>
<p>Friday</p>
